source/agora/network/Manager.d

```diff
@@ -79,6 +76,12 @@ public class NetworkManager
 
         /// Client
         NetworkClient client;
+
+        ///
+        public bool isValidator () const scope @safe pure nothrow @nogc
+        {
+            return this.key != PublicKey.init;
```

Comment:
at some point we shoould probably make this check if the peer is an active validators too.

Reply:
Yeah I'm thinking of changing it to check for the UTXO hash in the future, but it requires to add a dep to the Ledger.